arch/riscv/Kconfig

```diff
@@ -13,6 +13,14 @@ config COMPRESSED_ISA
 	bool
 	default y if 64BIT
 
+config RISCV_ATOMICS_ISA
```

Comment:
All these ISA-related configs will need to be refactored at some point, to be something like:

```
choice
	...
	default RISCV_ISA_RV64I if 64BIT
	default RISCV_ISA_RV32I

config RISCV_ISA_RV32I
	...

config RISCV_ISA_RV32E
	...

config RISCV_ISA_RV64I
	...

endchoice

config RISCV_ISA_EXT_M
	...

config RISCV_ISA_EXT_A
	...

config RISCV_ISA_EXT_G
	...
	imply RISCV_ISA_EXT_M
	imply RISCV_ISA_EXT_A
	imply RISCV_ISA_EXT_F
	imply RISCV_ISA_EXT_D
```